datastore/src/main/resources/application.properties

```diff
@@ -5,3 +5,5 @@ spring.datasource.password=
 #spring.datasource.driver-class-name=org.h2.Driver
 #spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=update
+application.baseUrl=http://c13422fcf88f.ngrok.io
```

Comment:
didnt we want to use an environment var here?

Reply:
Yes but Tuvia just figured out how to make it work in VSCode so this should hopefully never be an issue from this point forward